test/test-generate-request-details.js

```diff
@@ -233,7 +233,8 @@ suite('Test Generate Request Details', function() {
       headers: {
         'Topic': 'topic',
         'Urgency': 'urgency'
-      }
+      },
+      'proxy': 'proxy'
```

Comment:
A separate test should be added for the proxy option, this test is for the **Extra headers** parameter.